tools/topology/topology1/sof-rn-rt5682-rt1019.m4

```diff
@@ -0,0 +1,65 @@
+#
+# Topology for Renoir with I2S SP and DMIC.
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`acp-sp.m4')
+include(`acp-dmic.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include ACP DSP configuration
+include(`platform/amd/acp.m4')
+
+# Playback pipeline 1 on PCM 0 using max 2 channels of s16le.
+# Schedule 96 frames per 2000us deadline on core 0 with priority 0
+PIPELINE_PCM_ADD(sof/pipe-passthrough-playback.m4,
+        1, 0, 2, s16le,
+        2000, 0, 0,
+        48000, 48000, 48000)
+
+# playback DAI is ACPSP using 2 periods
+DAI_ADD(sof/pipe-dai-playback.m4, 1, ACPSP, 0, acp-headset-codec,
+PIPELINE_SOURCE_1, 2, s16le, 2000, 0, 0, SCHEDULE_TIME_DOMAIN_DMA)
+
+DAI_CONFIG(ACPSP, 0, 0, acp-headset-codec,
+	   ACPSP_CONFIG(I2S, ACP_CLOCK(mclk, 49152000, codec_mclk_in),
+                ACP_CLOCK(bclk, 3072000, codec_slave),
+                ACP_CLOCK(fsync, 48000, codec_slave),
+                ACP_TDM(2, 32, 3, 3),ACPSP_CONFIG_DATA(ACPSP, 0, 48000, 2)))
+
+# Capture pipeline 2 on PCM 0 using max 2 channels of s16le.
+PIPELINE_PCM_ADD(sof/pipe-passthrough-capture.m4,
+        2, 0, 2, s16le,
+        2000, 0, 0,
+        48000, 48000, 48000)
+
+# Capture DAI is ACPSP using 2 periods
+DAI_ADD(sof/pipe-dai-capture.m4, 2, ACPSP, 0, acp-headset-codec,
+PIPELINE_SINK_2, 2, s16le, 2000, 0, 0, SCHEDULE_TIME_DOMAIN_DMA)
```

Comment:
add tabs or something to indent the code please.

Reply:
Add

> add tabs or something to indent the code please.

Addressed the review comment.